### PR TITLE
Hero component

### PR DESCRIPTION
**Pull request by fatimaoday**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -11,9 +11,7 @@
   },
   "extends": [
     "eslint:recommended",
-    "plugin:react/recommended",
-    "plugin:prettier/recommended"
-  ],
+    "plugin:react/recommended"  ],
```
Comment (maqalaqil) [inline .eslintrc.json:14]: why is prettier removed here ?

File: src/App.js [hunk 2]
```diff
@@ -1,23 +1,9 @@
...
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div >
+      <HeroComponent/>
```
Comment (maqalaqil) [inline src/App.js:6]: can you please remove this since it wil cause merge conflicts

File: src/App.test.js [hunk 3]
```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+// import { render, screen } from '@testing-library/react';
+// import App from './App';
 
 test('renders learn react link', () => {
-  render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+  // render(<App />);
```
Comment (maqalaqil) [inline src/App.test.js:5]: why did you comment the test ? 
very disappointed in you removing the test? 😭

File: src/components/CtaButton.jsx [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+var buttonStyle = {
```
Comment (maqalaqil) [inline src/components/CtaButton.jsx:3]: don't use var

File: src/components/HeroComponent.css [hunk 5]
```diff
@@ -0,0 +1,55 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
```
Comment (maqalaqil) [inline src/components/HeroComponent.css:1]: add a folder for hero component

**Review by maqalaqil — CHANGES_REQUESTED** [review]
Overall good work @fatimaoday please check my comments and fix the changes

File: .eslintrc.json (hunk 1 above)
Comment (fatimaoday) [reply .eslintrc.json:14]: is was causing issues when running the app

File: src/App.js (hunk 2 above)
Comment (fatimaoday) [reply src/App.js:6]: removed 👍

File: src/components/CtaButton.jsx (hunk 4 above)
Comment (fatimaoday) [reply src/components/CtaButton.jsx]: changed to const 👍

File: src/components/HeroComponent.css (hunk 5 above)
Comment (fatimaoday) [reply src/components/HeroComponent.css]: done 👍

**Comment by fatimaoday** [comment]
here is a screenshot of the design when running the code.
Screenshot 2021-03-20 12 47 13

**Review by maqalaqil — APPROVED** [review]
LGTM

**Pull request merged**